packages/skeleton-svelte/src/lib/components/Listbox/types.ts

```diff
@@ -0,0 +1,60 @@
+import type { Snippet } from 'svelte';
+import type { HTMLAttributes, HTMLButtonAttributes } from 'svelte/elements';
+
+export interface ListboxContext {
+	readonly id: string;
+	readonly name: string;
+	readonly select: (value: string) => void;
+	readonly deselect: (value: string) => void;
+	readonly isSelected: (value: string) => boolean;
+}
+
+export interface ListboxProps extends HTMLAttributes<HTMLDivElement> {
+	/** Set the item name. */
+	name: string;
+	/** The value of the selected item(s). */
+	value?: string | string[];
+	/** Whether multiple items can be selected. */
+	multiple?: boolean;
+	/** Set the base classes. */
+	base?: string;
```

Comment:
Take a look at how we divide the various types of prop types. Just a line break + leading comment.
https://github.com/skeletonlabs/skeleton/blob/next/packages/skeleton-svelte/src/lib/components/Segment/types.ts#L19

 It helps readability a bit, or at least my old man brain :D

Reply:
Done!